### ISSUE TITLE
Add Suggested Comics

### ISSUE THREAD

**Issue by ArtskydJ** [ArtskydJ/comicsrss.com#75]
Something like this:
![suggested-comics](https://user-images.githubusercontent.com/1833684/40124145-7e9bf880-58ed-11e8-939e-f206621112d3.png)

The suggested comics pictures should be links to their respective preview pages.

### header img

Maybe I can scrape the header picture from the overview page, top-left corner

E.g. https://www.gocomics.com/foxtrot

```
div.layout-2col-sidebar.js-gc-lock-parent.js-gc-lock-parent--active > aside > div.gc-fade-bottom > picture
```

**Comment by ArtskydJ** [comment]
Here are some possible candidates:

- Calvin and Hobbes
- Foxtrot
- Peanuts
- Dilbert Classics
- Foxtrot Classics
- Peanuts Classics
- Garfield
- Pearls before Swine
- Garfield Classics

**Comment by ArtskydJ** [comment]
Move the search bar while I'm changing stuff anyway... That should fix #70.
